core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2571,6 +2572,23 @@ private[spark] object Utils extends Logging {
       sparkJars.map(_.split(",")).map(_.filter(_.nonEmpty)).toSeq.flatten
     }
   }
+
+  private[util] val REDACTION_REPLACEMENT_TEXT = "*********(redacted)"
+  private[util] val SECRET_REDACTION_PATTERN = "(?i)secret|password".r
```

Comment:
I did it work there... I tested here and UI and spark-submit already working. I think you can close this pull request and focus on #18802